index.php

```diff
@@ -493,3 +496,4 @@ function gtag(){dataLayer.push(arguments);}
 	_weirdDiv.setAttribute('class', 'weird-video-background');
 });
 </script>
+<script src="/trail.js" defer></script>
```

Comment:
Could we move this into the `static` folder to get the benefit of extra long cache time?